core/src/test/fixtures/android_app/src/main/java/com/novoda/gradle/release/test/SampleClass.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.gradle.release.test;
+
+public class SampleClass {
+
+    /**
+     * This is a main function, because reasons.
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
```

Comment:
This is just so that there's *some* source files for this to find when generating sources/javadoc.